vllm/config.py

```diff
@@ -148,9 +148,9 @@ def _verify_tokenizer_mode(self) -> None:
         self.tokenizer_mode = tokenizer_mode
 
     def _verify_sparsity(self) -> None:
-        supported_sparsity = ["sparse_w16a16"]
+        supported_sparsity = ["sparse_w16a16", "semi_structured_sparse_w16a16"]
```

Comment:
please rebase/merge with our main properly, it seems like you've picked up some recent changes into this diff